actionview/lib/action_view/cache_expiry.rb

```diff
@@ -42,12 +42,14 @@ def clear_cache
 
     private
       def dirs_to_watch
-        fs_paths = all_view_paths.grep(FileSystemResolver)
-        fs_paths.map(&:path).sort.uniq
-      end
-
-      def all_view_paths
-        ActionView::ViewPaths.all_view_paths.flat_map(&:paths)
+        fs_paths = ActionView::ViewPaths
+                     .all_view_paths
+                     .flat_map(&:paths)
+                     .grep(FileSystemResolver)
+                     .map!(&:path)
```

Comment:
We generally don't indent like this. It's usually just one line unless it's above > 130 or so.